Updated MVA lepton IDs

veelken opened the pull request:
- added HZZ electron ID from EGamma POG
- updated MVA lepton ID developed by ttH multilepton analysis

NOTE: the HZZ electron ID requires to pull code from the git repository of the EGamma POG:

cmsrel CMSSW_8_0_21
cd CMSSW_8_0_21/src
eval `scramv1 runtime -sh`
git cms-merge-topic ikrav:egm_id_80X_v2
git cms-merge-topic -u vhbb:vhbbHeppy80X
cd $CMSSW_BASE/external
cd slc6_amd64_gcc530/
git clone https://github.com/ikrav/RecoEgamma-ElectronIdentification.git data/RecoEgamma/ElectronIdentification/data
cd data/RecoEgamma/ElectronIdentification/data
git checkout egm_id_80X_v1
cd $CMSSW_BASE/src
scram b -j 16

Commit:
Merged egm_id_80X_v2 from repository ikrav

Commit:
Merged vhbbHeppy80X from repository vhbb

Commit:
Merge https://github.com/vhbb/cmssw into from-CMSSW_8_0_21

Commit:
added HZZ electron ID MVA

Commit:
updated lepton ID MVA used by ttH multilepton analysis to Summer16 (ReReco data and MC) training

arizzi commented:
how urgent is this stuff? we are about to push a button and the amount of stuff included here is not negligible possibly requiring another round of validation... (PS: are you subscribed to the vhbb heppy egroup?!?)

veelken commented:
Hi Andrea,

I would appreciate if you merge this PR as soon as possible, but I understand that you are about to start the production. It is fine for me if you the PR after starting the production (people working on ttH, H->tautau need to run a separate VHbb Ntuple production without the Njet >= 2 cut anyway).

Cheers,


Christian

P.S. I am subscribed to the vhbb heppy egroup

degrutto commented:
Hi @veelken 

here in this issues
https://github.com/vhbb/cmssw/issues/652
we were also  wondering if we better do not need to apply the
ele_mvaEleID_Trig_preselection(x)

to the Z-> ee electrons in case we use double electron triggers,
what it is your opinion?

Commit:
added L1Tau seeded single electron trigger and e+tau cross triggers recommended by tau trigger experts in yesterday's HTT meeting:
https://indico.cern.ch/event/598433/contributions/2475199/attachments/1413150/2162337/Davignon_TauTriggerRecommendations_HTTMeeting_15_02_17_v2.pdf
